composer.json

```diff
@@ -12,6 +12,8 @@
         "drupal/focal_point": "^2.1",
         "drupal/webp": "^1.0@RC",
         "drupal/media_entity_download": "^2.3",
-        "drupal/filefield_paths": "^1.0@beta"
+        "drupal/filefield_paths": "^1.0@beta",
+        "drupal/lazy": "3.x-dev",
```

Comment:
The `lazysizes` bower asset is required to install lazy?

Reply:
In the module README file, it says so, [Lazy](https://git.drupalcode.org/project/lazy), I only used composer command & installed it

Reply:
Yeah the library has to be installed via composer